lib/client.js

```diff
@@ -54,11 +55,22 @@ class Client {
     }
     this._options = options;
     if (!this._options.credentials) {
-      throw new Error("g11n-pipeline: missing 'credentials' " + Object.keys(Consts.exampleCredentials));
+      throw new Error("g11n-pipeline: missing 'credentials'"); // don't give specific contents
+      // could be CF or IAM.
     }
-    var missingField = utils.isMissingField(this._options.credentials, Object.keys(Consts.exampleCredentials));
-    if (missingField.length !== 0) {
-      throw new Error("g11n-pipeline: missing credentials fields: \"" + missingField.join(' ') + "\" - expected: " + Consts.exampleCredentialsString);
+    if(this._options.credentials.apikey) {
```

Comment:
if `apikey` is present, assume it is IAM.

Reply:
the assumption is correct